app/src/main/java/com/programmers/calculator/model/Operator.java

```diff
@@ -0,0 +1,40 @@
+package com.programmers.calculator.model;
+
+public enum Operator {
+    PLUS, MINUS, MULTIPLY, DIVIDE;
```

Comment:
enum에 상태를 갖게되면 아래 코드를 많이 개선할 수 있어요.
```suggestion
    PLUS('+'), MINUS('-'), MULTIPLY('*'), DIVIDE('/');
    
    private final char value;
```